chaosoci/compute/actions.py

```diff
@@ -1,26 +1,66 @@
 # -*- coding: utf-8 -*-
+from random import choice
+from typing import Any, List, Dict
+
 from oci.core import ComputeClient
+from oci.config import from_file
```

Comment:
I like sorting the imports if I can, so:

```
from oci.config import from_file
from oci.core import ComputeClient
```

Maybe we will enable isort some day to help :)

Reply:
I've installed isort, I will sort them all, I was thinking about this a few days back.